roles/st2/tasks/auth.yml

```diff
@@ -39,3 +39,18 @@
   notify:
     - restart st2api
     - restart st2stream
+
+- name: auth | Create root's CLI configuration directory
+  file: path=/root/.st2 state=directory
```

Comment:
Additionally, to be consistent with other playbooks in this repo, please use native multi-line yaml syntax (avoid `a=b`).
See: https://www.ansible.com/blog/ansible-best-practices-essentials for more info.